filament/include/filament/RenderableManager.h

```diff
@@ -464,7 +534,12 @@ class UTILS_PUBLIC RenderableManager : public FilamentAPI {
                 size_t offset = 0;
                 size_t count = 0;
             } morphing;
+            struct {
+                bool bonePairsSet = false;
```

Comment:
could we use `!bonePairs.empty()` instead?

Reply:
this is prepared for the possibility that some primitives of the renderable don't use skinning and others do. This allows to set an empty vector for no skinning geometries.

Reply:
Ok, I have removed it. We don't expect primitives without skinning defined together with primitives with skinning